src/Opdex.Platform.WebApi/OpenApi/Wallets/GetAddressBalanceByTokenOperationProcessor.cs

```diff
@@ -15,7 +15,7 @@ public bool Process(OperationProcessorContext context)
 
         var tokenAddressParameter = context.OperationDescription.Operation.Parameters.First(parameter => parameter.Name == "token");
         tokenAddressParameter.Schema.Example = "tF83sdXXt2nTkL7UyEYDVFMK4jTuYMbmR3";
-        tokenAddressParameter.Schema.MinLength = 30;
+        tokenAddressParameter.Schema.MinLength = 3;
```

Comment:
This is interesting because the frontend has been working fine with this validation, wondering if it was ever working 🤔

Reply:
This doesn't perform validation, it is used to produce the OAS doc

Reply:
aaahhhh